packages/demos/src/custom-logging/index.test.ts

```diff
@@ -18,7 +19,7 @@ describe('integration test: "custom logging"', () => {
   let consoleMessages: ConsoleMessage[];
 
   beforeAll(async () => {
-    server = await startServer(webpackConfigs);
+    server = await startServer(webpackConfigs, nodeWebpackConfig);
```

Comment:
Maybe you could also use an implicit detection via `target: 'node'` instead of this in my opinion unattractive API? This way the user would not have to be burdened with it.

Reply:
Hm, the `node` target is also used for the feature app commonjs bundles. I can’t think of a unique discrimitor.

Reply:
Also, there is either none or one node integegrator. Therefore a differently structured item in the list of webpack configs is not the best design, I think. It should be its own entity.

Reply:
By the way, maybe the special meaning would have been clearer if I named it `nodeIntegratorWebpackConfig` instead of `nodeWebpackConfig`?